docs/conf.py

```diff
@@ -54,9 +54,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'0.0.1'
+version = u'0.1'
```

Comment:
They should: the package version is 0.1 acording to [setup.py](https://github.com/MDAnalysis/pytng/blob/master/setup.py#L63) and therefore I changed it in the docs.